### PR TITLE
update tutorial

### PR DESCRIPTION
**Pull request by herissondev**
I had a bit of trouble to set up live_state on my phoenix app as there where some missing pieces on the tutorial. 
This Pr only adds a bit more details into how to setup live_state. 

Thanks for the work !

### CODE REVIEW

File: docs/tutorial_start.md [hunk 1]
```diff
@@ -66,30 +66,58 @@ To create our `<contact-form>` custom element, we'll use a generator to help us
...
 ```
 
-This will generate an element for us in `app/js/contact-form.ts`. While LiveState itself is not tied to any specific library, for the purposes of convenience we generate an element based on the [lit](https://lit.dev) library. The generator will also install the necessary npms for you.
+This will generate an element for us in `assets/js/contact-form.ts`. While LiveState itself is not tied to any specific library, for the purposes of convenience we generate an element based on the [lit](https://lit.dev) library. The generator will also install the necessary npms for you.
 
-We'll also want to add an import for our element in `app/js/custom_elements.js`. You'll want to create this file if it doesn't exit. Add this line for the import:
+We'll also want to add an import for our element in `assets/js/custom_elements.js`. You'll want to create this file if it doesn't exit. Add this line for the import:
 
 ```javascript
+// assets/js/custom_elements.js
```
Comment (superchris) [inline docs/tutorial_start.md:74]: Just curious about this commented out line

**Review by superchris — COMMENTED** [review]
This is great thank you soo much!! Just had a tiny question, and then will merge this.

File: docs/tutorial_start.md (hunk 1 above)
Comment (herissondev) [reply docs/tutorial_start.md:74]: It's redundant you're right

**Comment by herissondev** [comment]
Out of context, but i'd like to try and build an igniter installer if you don't mind.
[igniter](https://hexdocs.pm/igniter/readme.html)

**Pull request merged**